aries-site/src/examples/templates/filtering/FilteringCards.js

```diff
@@ -0,0 +1,607 @@
+        };
+        setFilters(nextFilters);
+      }}
+    />
+  </FormField>
+);
+
+LocationFilter.propTypes = {
+  filters: PropTypes.shape({
+    role: PropTypes.oneOfType([PropTypes.func, PropTypes.bool]),
+    status: PropTypes.oneOfType([PropTypes.func, PropTypes.bool]),
+  }).isRequired,
+  setFilters: PropTypes.func.isRequired,
+  location: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.arrayOf(PropTypes.string),
+  ]).isRequired,
+  setLocation: PropTypes.func.isRequired,
+};
+
+const HoursAvailableFilter = ({
+  hoursAvailable,
+  setHoursAvailable,
+  filters,
+  setFilters,
+}) => {
+  return (
+    <Box flex={false}>
+      <FormField label="Remaining Available Work Hours" pad="medium">
+        <Stack>
+          <Box background="border" height="3px" direction="row" />
+          <RangeSelector
+            name="range-selector-employee-count"
+            id="range-selector-employee-count"
+            min={defaultHoursAvailable[0]}
+            max={defaultHoursAvailable[1]}
+            values={hoursAvailable}
+            onChange={nextRange => {
+              setHoursAvailable(nextRange);
+              const nextFilters = {
+                ...filters,
+                hoursAvailable: nexthoursAvailable =>
+                  nexthoursAvailable >= hoursAvailable[0] &&
+                  nexthoursAvailable <= hoursAvailable[1],
+              };
+              setFilters(nextFilters);
+            }}
+          />
+        </Stack>
+      </FormField>
+      <Text size="small">
+        {`${hoursAvailable[0]} - ${hoursAvailable[1]} hours`}
+      </Text>
+    </Box>
+  );
+};
+
+HoursAvailableFilter.propTypes = {
+  filters: PropTypes.shape({
+    role: PropTypes.oneOfType([PropTypes.func, PropTypes.bool]),
+    status: PropTypes.oneOfType([PropTypes.func, PropTypes.bool]),
+  }).isRequired,
+  setFilters: PropTypes.func.isRequired,
+  hoursAvailable: PropTypes.arrayOf(PropTypes.number).isRequired,
+  setHoursAvailable: PropTypes.func.isRequired,
+};
+
+const RecordSummary = ({ data, filtering }) => (
+  <Box direction="row" gap="xxsmall">
+    <Text color="text-weak" size="small" weight="bold">
+      {data.length}
+    </Text>
+    <Text color="text-weak" size="small">
+      {filtering ? 'results of ' : 'items'}
+    </Text>
+    {filtering && (
+      <Box direction="row" gap="xxsmall">
+        <Text color="text-weak" size="small" weight="bold">
+          {allData.length}
+        </Text>
+        <Text color="text-weak" size="small">
+          items
+        </Text>
+      </Box>
+    )}
+  </Box>
+);
+
+RecordSummary.propTypes = {
+  data: PropTypes.arrayOf(
+    PropTypes.shape({
+      role: PropTypes.string,
+      status: PropTypes.string,
+    }),
+  ).isRequired,
+  filtering: PropTypes.bool.isRequired,
+};
+
+const Results = ({ data }) => {
+  const size = useContext(ResponsiveContext);
+
+  return (
+    <Grid columns={size !== 'small' ? 'medium' : '100%'} gap="medium">
```

Comment:
Related to: https://github.com/hpe-design/design-system/pull/1084/files?file-filters%5B%5D=.mdx#r485976169

Could the columns here be:
```suggestion
    <Grid columns={size !== 'small' ? 'small' : 'xsmall'} gap="medium">
```